services/scripts.py

```diff
@@ -1,43 +1,50 @@
 # Music recommendation application
-import csv
-
+import pandas as pd
+from collections import defaultdict
 from cfg import TableIndexes
```

Comment:
```suggestion
import pandas as pd

from collections import defaultdict

from cfg import TableIndexes
```
just a cosmetic change